table/init.go

```diff
@@ -79,9 +79,7 @@ func SetCsCapacity(capacity int) {
 	csCapacity = capacity
 }
 
-func init() {
-	// fibTableAlgorithm = core.GetConfigStringDefault("tables.fib.algorithm", "nametree")
-	fibTableAlgorithm = "hashtable"
+func CreateFIBTable(fibTableAlgorithm string) {
```

Comment:
@zjkmxy does this look like a good approach?

Reply:
Yes! That's what I meant.